modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -39,15 +39,15 @@ spec:
           pod: {}
         - name: bridge-net <1>
           multus:
-            networkName: <network-namespace/a-bridge-network> <2>
+            networkName: default/<a-bridge-network> <2>
 ...
 ----
 <1> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
 <2> The name of the network attachment definition.
 +
 [NOTE]
 ====
-The example virtual machine is connected to both the `default` pod network and `bridge-net`, which is defined by a network attachment definition named `a-bridge-network`. If the network attachment definition is not in the same namespace as the VM, you must combine the namespace and the network name to create the network attachment definition name.
+The example virtual machine is connected to both the default pod network and the secondary network defined by a network attachment definition (NAD). The NAD must be created either in the same namespace as the VM or in the `default` namespace. However, if the NAD is defined in the `default` namespace, it must be prefixed such as `default/a-bridge-network`.
```

Comment:
This is almost accurate. If the NAD is to be created in the same namespace as the VM, but the current working namespace is different, then the target namespace should also be prefixed (same thing with the VM, as with all other namespaced Kubernetes resources).

Reply:
I would replace this sentence:
_However, if the NAD is defined in the `default` namespace, it must be prefixed such as `default/a-bridge-network`._
with something like
_However, if the current working namespace is different than the one where the NAD is created, the NAD name must be prefixed with the NAD namespace, e.g.  `default/a-bridge-network` (as in any other namespaced resource in Openshift)._